source/technical/sharing.rst

```diff
@@ -259,3 +259,58 @@ Next, you can navigate back to the original artifact URL you were given. The
 
 After the duration for the invite has passed, you will be automatically removed
 from the project.
+
+
+Best Practices
+==============
+
+In this section, we describe some of the best practices when packaging
+experiments.
+
+Things to Do
+------------
+
+Provide a background of your experiment in a succient paragraph. Provide any
+relevant citations to credit original authors.
+
+Seperate your notebook into parts that make sense. For example, have one file
+that reproduces the experiment, and another file that runs the analysis. This
+allows replicators to choose what parts they want to run. It can be helpful to
+have a clear file where a user should start, such as a README, that gives an
+overview of your notebook.
+
+Give an outline of the features of your notebook, explaining the purpose of
+each file.
+
+In each file, explain what the methodology is, and the preconditions. If your
```

Comment:
The preconditions seem more important to call out here:

"In each notebook, make sure to **call out any preconditions**. If your..."